aries-site/src/examples/templates/Card/Card.js

```diff
@@ -41,6 +43,14 @@ const DesignSystemCard = ({
             align="start"
             pad={adjustPad(direction, 'header', theme)}
           >
+            {avatar && (
+              <Avatar
```

Comment:
```suggestion
              <Avatar
                 flex={false}
```
I've found this is often needed so it doesn't get squished on certain screen widths.